src/um/d3d12.rs

```diff
@@ -4128,6 +4275,23 @@ interface ID3D12GraphicsCommandList5(ID3D12GraphicsCommandList5Vtbl)
         shadingRateImage: *mut ID3D12Resource,
     ) -> (),
 }}
+RIDL! {#[uuid(0xc3827890, 0xe548, 0x4cfa, 0x96, 0xcf, 0x56, 0x89, 0xa9, 0x37, 0x0f, 0x80)]
+interface ID3D12GraphicsCommandList6(ID3D12GraphicsCommandList6Vtbl)
```

Comment:
They forgot about this in the official docs https://learn.microsoft.com/en-us/windows/win32/api/d3d12/ :slightly_smiling_face: :

![image](https://user-images.githubusercontent.com/2325264/202693368-e070a896-206b-4cf8-a10a-14a4d1c92337.png)